src/test/kotlin/com/makarytskyi/rentcar/services/RepairingServiceTest.kt

```diff
@@ -5,14 +5,15 @@ import com.makarytskyi.rentcar.repository.CarRepository
 import com.makarytskyi.rentcar.repository.RepairingRepository
 import com.makarytskyi.rentcar.service.impl.RepairingServiceImpl
 import fixtures.CarFixture.randomCar
```

Comment:
```suggestion
import com.makarytskyi.rentcar.fixtures.CarFixture.randomCar
```
you can fix this in separate PR